docs/rst/manual/reference/configuration/magic_strings.rst

```diff
@@ -0,0 +1,259 @@
+.. topic:: CONFIG_PATH
+
+   Will be replaced with the path to the directory containing the config file for the experiment.
+
+
+   *Example:*
+
+   ::
+
+      YOUR_WORKFLOW_JOB <CONFIG_PATH>
+
+   Will be replaced with the following at runtime:
+
+   ::
+
+      YOUR_WORKFLOW_JOB /path/to/config/
+
+.. _ms_config_file:
+.. topic:: CONFIG_FILE
+
+   Will be replaced with the name of the config file for the experiment.
+
+
+   *Example:*
+
+   ::
+
+      YOUR_WORKFLOW_JOB <CONFIG_FILE>
+
+   Will be replaced with the following at runtime:
+
+   ::
+
+      YOUR_WORKFLOW_JOB example.ert
+
+.. _ms_config_file_base:
+.. topic:: CONFIG_FILE_BASE
+
+   Will be replaced with the base name of the config file for the experiment.
+   This is the filename without the extension.
+
+
+   *Example:*
+
+   ::
+
+      YOUR_WORKFLOW_JOB <CONFIG_FILE_BASE>
+
+   Given that the filename is `example.ert`, this will be replaced with the following at runtime:
+
+   ::
+
+      YOUR_WORKFLOW_JOB example
+
+.. _ms_cwd:
+.. topic:: CWD
+
+   Will be replaced with the current working directory.
+   This will usually be the same as the :ref:`CONFIG_PATH <ms_config_path>`.
+
+
+   *Example:*
+
+   ::
+
+      YOUR_WORKFLOW_JOB <CWD>
+
+   Will be replaced with the following at runtime:
+
+   ::
+
+      YOUR_WORKFLOW_JOB /data/repos/ert/test-data/local/poly_example
+
+.. _ms_exp_variables:
+
+Experiment Variables
+--------------------
+
+.. warning::
+   The experiment variables cannot be used with workflows, only forward model jobs!
+
+.. _ms_iens:
+.. topic:: IENS
+
+   Will be replaced with the index for the current realization.
+   The realization number uses zero-based indexing.
+
+
+   *Example:*
+
+   ::
+
+      SIMULATION_JOB some_forward_model_job <IENS>
+
+   Will in the third realization be replaced with the following at runtime:
+
+   ::
+
+      SIMULATION_J some_forward_model_jobOB 2
+
+.. _ms_iter:
+.. topic:: ITER
+
+   Will be replaced with the current iteration number.
+   The iteration number uses zero-based indexing.
+
+
+   *Example:*
+
+   ::
+
+      SIMULATION_JOB some_forward_model_job <ITER>
+
+   In the second iteration of your experiment, this will be replaced with the following at runtime:
+
+   ::
+
+      SIMULATION_J some_forward_model_jobOB 1
+
+.. _ms_eclbase:
+.. topic:: ECLBASE / ECL_BASE
+
+   Will be replaced by the basename for the ECLIPSE simulations generated by ERT.
+   See :ref:`ECLBASE <eclbase>` for more details.
+   For none-Eclipse jobs this will be replaced by the job name (with an iteration identifier)
+
+
+   *Example:*
+
+   ::
+
+      SIMULATION_JOB some_forward_model_job <ECLBASE>
+
+   Given that the ECLBASE has been specified using `ECLBASE eclipse/model/MY_VERY_OWN_OIL_FIELD-%d`
+   in the config file, this will be replaced with the following in the second iteration:
+
+   ::
+
+      SIMULATION_JOB some_forward_model_job eclipse/model/MY_VERY_OWN_OIL_FIELD-2
+
+
+.. _ms_geoid:
+.. topic:: GEOID
```

Comment:
It is only by using the `BatchSimulator` that `geoid` actually becomes resolved and hence it is only available in Everest, not in ERT. Perhaps we should remove it again from the documentation here? 🤔